httpmw/ip_cognito_auth.go

```diff
@@ -193,24 +192,41 @@ func IpCognitoAuth(ipRange string, svc cognitoidentityprovideriface.CognitoIdent
 			return
 		}
 
-		model, ok := ch.Get(token)
+		data, err := cache.Get(c, token).Bytes()
```

Comment:
We should probably make cache key something like `fmt.Sprintf("access_token:%s", token)` so we have at least a little bit of namespaceng for our keys.

Reply:
Will let @Memije change the key name in his PR